src/bin/utils/sync.ts

```diff
@@ -0,0 +1,179 @@
+import { confirm } from '@inquirer/prompts';
+
+import { generateFieldId } from '@/src/bin/utils/schema';
+import type { Schema } from '@/src/types/schema';
+
+/**
+ * Fetch the available spaces for the current session.
+ *
+ * @param sessionToken The session token to use for the request.
+ *
+ * @returns The available spaces for the current session.
+ */
+export const getSpaces = async (
+  sessionToken: string,
+): Promise<{ id: string; handle: string; name: string }[]> => {
+  let res;
+  try {
+    res = await fetch('https://ronin.supply/-/ronin/spaces', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${sessionToken}`,
+      },
+    });
+  } catch (err) {
+    throw new Error('Failed to fetch available spaces: ${err.message}');
+  }
+
+  const text = await res.text();
+
+  let json;
+  try {
+    json = JSON.parse(text);
+  } catch (err) {
+    throw new Error(`Failed to parse response from server: ${text}`);
+  }
+
+  return json.spaces;
+};
+
+/**
+ * Fetch the existing schemas for the configured space.
+ *
+ * @param token Either an App Token or a Session Token.
+ * @param space The ID of the space to fetch schemas for.
+ *
+ * @returns The schemas of the provided space
+ */
```

Comment:
```suggestion
/**
 * Fetch the existing schemas for the configured space.
 *
 * @param token Either an app token or a session token.
 * @param space The ID of the space to fetch schemas for.
 *
 * @returns The schemas of the provided space.
 */
```

I get the intention, but we don't normally capitalize those two.